packages/truffle-debugger/lib/debugger.js

```diff
@@ -44,25 +43,64 @@ export default class Debugger {
    * @return {Debugger} instance
    */
   static async forTx(txHash, options = {}) {
-    expect.options(options, [
-      "contracts",
-      "provider"
-    ]);
+    expect.options(options, ["contracts", "provider"]);
 
     let session = new Session(
-      options.contracts, options.files,
-      txHash, options.provider
+      options.contracts,
+      options.files,
+      options.provider,
+      txHash
     );
 
     try {
       await session.ready();
+      debug("session ready");
     } catch (e) {
-      throw e;
+      debug("error occurred, unloaded");
```

Comment:
Maybe worth adding `unloaded transaction` or even `unloaded transaction (if any)`? What do you think? I understand that the `(if any)` would be more correct, but it might make it more confusing. Not sure - might be worth just keeping as-is for simplicity.

Reply:
Its just a debug message, I don't think it matters.